.github/workflows/pylint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         sudo apt-get -y install libbz2-dev
         python -m pip install --upgrade pip
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-        pip install pylint
+        pip install pylint==2.12.2
```

Comment:
why do we need to fix the pylint version? Is it already to prevent installing early 3.0 versions (that are partly backwards incompatible I guess), why then not fix it to 2.14.0

Reply:
there seems to be a change in the way new pylint version works that made it fail the test on the commit `update second half of the test suite`

Reply:
it might be that they changed that version to 3.0 alpha (at least they mention on their github that all incompatible changes were put in 3.0a), but that's anyway not the point of this PR, so can be adapted later